examples/readme-example/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "example-readme"
```

Comment:
I think this should follow the convention set by the other examples. So the directory is called `readme` while the crate is called `example-readme`.